apps/web/lib/apps/installed/[category]/getServerSideProps.tsx

```diff
@@ -1,78 +0,0 @@
-import type { GetServerSidePropsContext } from "next";
-import { z } from "zod";
-
-import { AppCategories } from "@calcom/prisma/enums";
-
-export type querySchemaType = z.infer<typeof querySchema>;
-
-export const querySchema = z.object({
-  category: z.nativeEnum(AppCategories),
-});
-
-export async function getServerSideProps(ctx: GetServerSidePropsContext) {
-  // get return-to cookie and redirect if needed
-  const { cookies } = ctx.req;
-
-  const returnTo = cookies["return-to"];
-
-  if (cookies && returnTo) {
-    ctx.res.setHeader("Set-Cookie", "return-to=; path=/; expires=Thu, 01 Jan 1970 00:00:00 GMT");
-    const redirect = {
-      redirect: {
-        destination: `${returnTo}`,
-        permanent: false,
-      },
-    } as const;
-
-    return redirect;
-  }
```

Comment:
this is handled in `middleware`.
<img width="1255" alt="Screenshot 2025-01-24 at 10 07 43 AM" src="https://github.com/user-attachments/assets/b2e280c2-ffae-4a8e-a9d5-6d9713b4655b" />

Reply:
you can test this cookie by
(1) going to http://localhost:3000/getting-started/connected-calendar
(2) press "Connect" with some app (this sets a cookie with `return-to`)
(3) and then go to http://localhost:3000/apps/installed

You will return to http://localhost:3000/getting-started/connected-calendar